packages/core/index.mjs

```diff
@@ -1,8 +1,10 @@
 import zapier from './src/index.js';
 import packageJson from './package.json' with { type: 'json' };
 import _tools from './src/tools/exported.js';
+import consoleSingleton from './src/tools/console-singleton.js';
 zapier.version = packageJson.version;
 zapier.tools = _tools;
+zapier.console = consoleSingleton;
```

Comment:
Doesn't `import { console } from 'zapier-platform-core';` now get you an object that has `.get()` and `.initialize()` methods?

I think I now understand your earlier version of `console-singleton.js`.

Perhaps we can use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Proxy to from `console-singleton.js` export a proxy for the global `console` object, with a `get(target, prop, receiver) { .. }` that at run-time checks if `loggerConsole` is set and depending on that either proxies to that or else falls back to the global one?